meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/files/0009-BootChainDxe-fix.patch

```diff
@@ -0,0 +1,15 @@
+diff --git edk2-tegra.a/edk2-nvidia/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxe.c edk2-tegra.b/edk2-nvidia/Silicon/NVIDIA/Drivers/BootChainDxe/BootChainDxe.c
```

Comment:
@maciej-narojczyk not sure if you are following the progress in https://github.com/OE4T/meta-mender-community/pull/19#discussion_r1676842714 but I'm curious if you could shed any light there on the necessity/purpose of this patch and `0007-runtime-access-for-KernelCommandLine-efivar.patch`